### DISCUSSION TITLE
Show encryption type when running "info"

### DISCUSSION THREAD

**Discussion by AndresGomezGH** [pdfcpu/pdfcpu#942] [Ideas]
Hi! First of all, I love the tool, thanks for your work! Well, about the new feature, I would like to know if it is possible to have pdfcpu display the encryption type when using the "info" command. Example of the section:

━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
           Encrypted: Yes
Type: Standard V5 R5 256-bit AES
         Permissions:
permission bits: 000000000000 (x000)
Bit  3: false (print(rev2), print quality(rev>=3))
Bit  4: false (modify other than controlled by bits 6,9,11)
Bit  5: false (extract(rev2), extract other than controlled by bit 10(rev>=3))
Bit  6: false (add or modify annotations)
Bit  9: false (fill in form fields(rev>=3)
Bit 10: false (extract(rev>=3))
Bit 11: false (modify(rev>=3))
Bit 12: false (print high-level(rev>=3))

**Comment by hhrutter** [comment]
While it’s technically possible to implement, I am personally against it.

Although one could argue that providing encryption details requires knowledge of the password, meaning only those with rightful access to the file could view this information, the actual details of the encryption technique are relevant to only a small group of people—some of whom may have malicious intentions. 

We don't want to offer any potential vectors for hacking as part of the output in the `pdfcpu info` command.

Having said that, you are free to implement this feature in your own fork.

Thank you for using pdfcpu 💚

**Reply by AndresGomezGH** [reply]
Hi Horst! Thanks for the explanation, I understand your motivations. I will keep the suggestion in mind. Best regards!
